### PR TITLE
Adds Braga City Guide page

### PR DESCRIPTION
**Pull request by frm**
Has more pictures than the initial design.

Also swaps the widder pictures to cropped versions, since they were small on mobile.

Desktop:

screen shot 2015-08-14 at 18 32 49

Picture swapping:

screen shot 2015-08-14 at 18 33 42

screen shot 2015-08-14 at 18 33 22

### CODE REVIEW

File: app/_includes/braga/00_main.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="u-paddingLarge"></div>
+
+<section class="CityGuide">
+  <h2 class="SectionTitle">Braga City Guide</h2>
+
+  <div class="CityGuide-intro">
+    <div class="CityGuide-row">
+        <div class="CityGuide-description CityGuide-text">
```
Comment (azevedo) [inline app/_includes/braga/00_main.html]: Having two descendants in the same element is a smell. We actually don't allow it ;)

**Comment by azevedo** [comment]
Is this project following SuitCSS?

File: app/_includes/braga/00_main.html (hunk 1 above)
Comment (frm) [inline app/_includes/braga/00_main.html]: Fixed that now, on multiple files.

**Comment by zamith** [comment]
@azevedo-252 I believe it was supposed to be at least similar to that, and I'd say that if we can get there in a few changes that is great, but if that's a big effort, it is not worth it (in this PR anyway).

**Comment by azevedo** [comment]
@zamith I agree with you I was just trying to establish what I'm supposed to expect :)

**Comment by frm** [comment]
@zamith @azevedo-252 I refactored the CSS into smaller components. I believe it's closer to SuitCSS now. At least it feels a lot more cleaner.

**Comment by nunomsantos** [comment]
Hi there. I noticed there's a relative URL on the [Venue page](https://github.com/subvisual/rubyconfpt-2015/blob/master/app/_includes/venue/20_how_to_get_here.html#L34) that should be absolute. Maybe you could fix it in this PR? Have a nice conference. Cheers.

**Comment by zamith** [comment]
@nunomsantos Thanks, we had noticed it before, just didn't go around to fixing it. This is a reminder that we should.

**Comment by zamith** [comment]
:+1:, we can refactor this later, if need be.

**Pull request merged**